libraries/bot-azure/src/test/java/com/microsoft/bot/azure/AzureQueueTests.java

```diff
@@ -97,28 +105,31 @@ public void continueConversationLaterTests() {
 
         QueueMessageItem messages = queue.receiveMessage();
         JacksonAdapter jacksonAdapter = new JacksonAdapter();
-        String messageJson = messages.getMessageText();
+        String messageJson = new String(Base64.decodeBase64(messages.getMessageText()));
         Activity activity = null;
+
         try {
             activity = jacksonAdapter.deserialize(messageJson, Activity.class);
         } catch (IOException e) {
             e.printStackTrace();
             Assert.fail();
         }
-        Assert.assertTrue(activity.isType(ActivityTypes.EVENT));
+
+        Assert.assertEquals(ActivityTypes.EVENT, activity.getType());
```

Comment:
```suggestion
        Assert.assertTrue(activity.isType(ActivityTypes.EVENT));
```

As Tracy mentioned in some PRs, we should use `isType`